aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts

```diff
@@ -65,38 +69,103 @@ const failureMetrics = modelIds.map((modelId, index) => ({
   ),
 }));
 
-export const chatCompletionJobExecutionHighFailureRateConfiguration = {
-  AlarmName: 'genai_chat_completion_job_execution_high_failure_rate',
-  AlarmDescription: 'genai_chat_completion_job_execution_high_failure_rate',
-  ActionsEnabled: true,
-  OKActions: [],
-  AlarmActions: [SNS_TOPIC],
-  InsufficientDataActions: [],
-  EvaluationPeriods: 5,
-  DatapointsToAlarm: 5,
-  Threshold: 10,
-  ComparisonOperator: ComparisonOperator.GreaterThanThreshold,
-  TreatMissingData: 'missing',
-  Metrics: [
-    {
-      Id: 'failure_rate',
-      Label: 'failure_rate',
-      ReturnData: true,
-      Expression: '100*(failures/total)',
-    },
-    {
-      Id: 'failures',
-      Label: 'failures',
-      ReturnData: false,
-      Expression: 'SUM([m6, m7, m8, m9, m10])',
-    },
-    ...failureMetrics,
-    {
-      Id: 'total',
-      Label: 'total_jobs',
-      ReturnData: false,
-      Expression: 'SUM([m1, m2, m3, m4, m5])',
-    },
-    ...startMetrics,
-  ],
-};
+export const chatCompletionJobExecutionHighFailureRateConfiguration: PutMetricAlarmInput =
+  {
+    AlarmName: 'genai_chat_completion_job_execution_high_failure_rate',
+    AlarmDescription:
+      'Alarms when chat completion jobs are experiencing a high failure rate.',
+    ActionsEnabled: true,
+    OKActions: [],
+    AlarmActions: [SNS_TOPIC],
+    InsufficientDataActions: [],
+    EvaluationPeriods: 5,
+    DatapointsToAlarm: 5,
+    Threshold: 10,
+    ComparisonOperator: ComparisonOperator.GreaterThanThreshold,
+    TreatMissingData: 'missing',
+    Metrics: [
+      {
+        Id: 'failure_rate',
+        Label: 'failure_rate',
+        ReturnData: true,
+        Expression: '100*(failures/total)',
+      },
+      {
+        Id: 'failures',
+        Label: 'failures',
+        ReturnData: false,
+        Expression: 'SUM([m6, m7, m8, m9, m10])',
+      },
+      ...failureMetrics,
+      {
+        Id: 'total',
+        Label: 'total_jobs',
+        ReturnData: false,
+        Expression: 'SUM([m1, m2, m3, m4, m5])',
+      },
+      ...startMetrics,
+    ],
+  };
+
+const browserIndices = BROWSERS.map((_, i) => i + 1);
+
+// TODO: Unify this code with dashboard metrics if possible
```

Comment:
This alarm more or less uses the same metrics as the equivalent dashboard widget, but alarms and dashboard syntax are unfortunately a bit different so it's a bit tricky to reuse code cleanly. I'm sure we can factor stuff out and organize this in a more modular fashion, but I figured that's something we can do as a follow-up.